payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -138,6 +142,20 @@ internal class ElementsSessionJsonParser(
         return cardBrandChoice.optBoolean(FIELD_ELIGIBLE, false)
     }
 
+    private fun parseLinkFlags(json: JSONObject): Map<String, Boolean> {
```

Comment:
Do we need to store this as a map? I fear this might break the entire elements/sessions response if we change from string -> boolean to a nested object or something.